.github/workflows/secrel.yml

```diff
@@ -13,7 +13,10 @@ on:
   workflow_dispatch:
 
 env:
-  GITHUB_ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+  # If SecRel can't push signed images, revert to use `secrets.ACCESS_TOKEN` instead.
+  # secrets.ACCESS_TOKEN_GRADLE_BUILD only has `read:packages` permission
+  # while secrets.ACCESS_TOKEN has `repo` and `write:packages` permission
+  GITHUB_ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN_GRADLE_BUILD }}
```

Comment:
@sethdarragile6 FYI

Reply:
Once SecRel stop failing, we should test this without Gradle caching to ensure this `ACCESS_TOKEN_GRADLE_BUILD` is sufficient and then remove the comment and `secrets.ACCESS_TOKEN` from https://github.com/department-of-veterans-affairs/abd-vro-internal/settings/secrets/actions